### PR TITLE
[TASK] Also allow newlines before the `?` operator

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -138,7 +138,11 @@
     <rule ref="PEAR.WhiteSpace.ObjectOperatorIndent"/>
     <rule ref="PEAR.WhiteSpace.ScopeClosingBrace"/>
     <rule ref="Squiz.WhiteSpace.CastSpacing"/>
-    <rule ref="Squiz.WhiteSpace.LogicalOperatorSpacing"/>
```
Comment (JakeQZ) [inline phpcs.xml:141]: Looks like the wrong rule is replaced.

File: phpcs.xml [hunk 2]
```diff
@@ -138,7 +138,11 @@
...
+    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
+        <properties>
+            <property name="ignoreNewlines" value="true"/>
+        </properties>
+    </rule>
     <rule ref="Squiz.WhiteSpace.OperatorSpacing"/>
```
Comment (JakeQZ) [inline phpcs.xml:146]: It should be this one being replaced.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
The wrong rule has been replaced - `LogicalOperatorSpacing` should be left unchanged (though I'm not sure what it does, it doesn't seem to be documented).

File: phpcs.xml (hunk 2 above)
Comment (oliverklee) [reply phpcs.xml:146]: Good catch! Done.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**